connector/connectionFields.xml

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <connection-fields>
+
```

Comment:
Screenshot of the connection dialogue [here](https://docs.google.com/document/d/1vt6f0c2T74jgb_QqLFM8FEM12dh56BfD2Qh5Gw6rxWc/edit#bookmark=id.oc2z3uwg2ulz)